app/assets/stylesheets/govuk_frontend_rails.scss

```diff
@@ -12,6 +12,7 @@ $govuk-image-url-function: 'font-url';
 
 @import "components/flash";
 @import "components/currency_input";
+@import "components/a11y-dialog";
```

Comment:
This stops being `govuk_frontend_rails` now, I think. Does it make sense to rename it?

Reply:
I've added a commit that extracts our application specific components out to a separate `govuk_frontend_custom.scss`